pykafka/producer.py

```diff
@@ -17,17 +17,25 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 """
-__all__ = ["Producer", "AsyncProducer"]
+__all__ = ["Producer", "SynchronousProducer"]
```

Comment:
Yeah, I think that's a great idea. If we're in the real of making breaking changes, that'd make things overall a lot cleaner.